Ajuste de pré requisito para integracao com QR (#1537)
mercadopago/devsite-docs: guides/in-person-payments/qr-code/pre-requisites.en.md

| Point of sale (POS) | A **place to perform a transaction** on a store or physical shop. Each POS will be linked with a unique QR code. |
| Order | A purchase made by your client. Contains a list of products with an associated cost. |

> 

Reviewer: ```suggestion
```

>
> Note
>

> > In case of POS integrated with several tenants, see topic 4 and 5. Pay attention to the security of your integration and implement OAUTH.

Reviewer: ```suggestion
> In case of POS integrated with several tenants, see topic 4 and 5. Pay attention to the security of your integration and implement OAuth.
```

Once the test users are created, you can start with the integration and create the Stores and Point of Sale.

### 4. How to collect Access Token (OAUTH)

For you integrator, who will work with several stores that use Mercado Pago digital wallet, we advise you to do the OAUTH process - authentication between accounts, this process consists of the customer allowing their data to be shared with a third party system in a secure way.

The access token cannot be shared in any way other than OAUTH. [More information](https://www.mercadopago.[FAKER][URL][DOMAIN]/developers/en/guides/resources/credentials)

> 

Reviewer: ```suggestion
```

### 5. Cómo identificar tu integración

> To identify the orders that are transacted by your management system, include the sponsor ID, see in the [face-to-face order APIs](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/reference) how you will send this information.

Reviewer: ```suggestion
To identify the orders that are transacted by your management system, include the sponsor ID, see in the [instore orders APIs](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/reference) how you will send this information.
```

 
Step by step:

> > * Create an account on the Mercado Pago portal (Integrator identification)

Reviewer: ```suggestion
> * Create an account on the Mercado Pago portal (Integrator identification).
```

> > * Collect Collector ID (Cust ID or User ID) of your account

Reviewer: ```suggestion
> * Collect Collector ID (Cust ID or User ID) of your account.
```

> > * Include the Collector ID of your Integrator account inside the Sponsor ID of the seller

Reviewer: ```suggestion
> * Include the Collector ID of your Integrator account inside the Sponsor ID of the seller.
```

> > * Submit Sponsor ID on all QR transactions [see api](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/reference/instore_orders/_mpmobile_instore_qr_user_id_external_id/post)

Reviewer: ```suggestion
> * Submit Sponsor ID on all QR transactions. [See api](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/reference/instore_orders/_mpmobile_instore_qr_user_id_external_id/post).
```

> 

Reviewer: ```suggestion
```
